homework-5/hw5-task2.py

```diff
@@ -0,0 +1,9 @@
+i = 1
+while i <= 9:
+    j = 1
+    while j <= i:
+        multiplication = i * j
+        print(f"{j} * {i} = {multiplication}", end=" ")
+        j += 1
+    print()
+    i += 1
```

Comment:
მეოთხე დავალებაში ნაძვის ხეზე გიგზავნი ამოხსნას:
n = int(input('Enter a natural number below 50: '))

if not 0 < n < 50:
    print('Entered number does not meet the requirement.')
    exit(1)

for i in range(1, n):
    print(' ', end = '')
print('*')

for i in range(1, n):
    for j in range(1, n + i + 1):
        if j < n - i:
            print(' ', end='')
        elif j < n:
            print('/', end='')
        elif j > n:
            print('\\', end='')
        else:
            print('|', end='')
    print()

for i in range(1, n):
    print(' ', end = '')
print('|')